src/recipe/test/recipe.controller.spec.ts

```diff
@@ -57,29 +63,38 @@ describe('RecipeController', () => {
     });
   });
 
-  describe('FetchRecipe', () => {
+  describe('FindRecipe', () => {
     it('should fetch recipe given id', async () => {
       //given
-      const id = faker.number.int();
+      const recipeId = faker.number.int();
+      const userId = faker.number.int();
 
       //when
-      const fetchedRecipe = await recipeController.fetchRecipe(id);
+      const fetchedRecipe = await recipeController.fetchRecipe(
```

Comment:
in describe block you have `find` but in controller it's `fetch` - unify this (for me it doesn't matter which one you'll pick)